src/lib/components/About.svelte

```diff
@@ -0,0 +1,61 @@
+<article>
+  <section>
+    <h3>Hi, we are MISC 👋</h3>
+
+    <p>私たちは名電高校の情報システム部です。</p>
+    <p>
+      Web サイトやシステム制作やゲーム制作といったプログラミング、<br />
+      IT パスポート試験や基本情報技術者試験などに向けた資格勉強をしています。
+    </p>
+    <p>
+      また映像・音楽・3D モデリング、 3D
+      プリンターやレーザーカッターを用いた作品を制作したりもします。
+      個々の活動を主としつつ、コンテストや発表に向けて
+      グループで活動することもあります。
+    </p>
+
+    <p>活動場所は北校舎 3 階の 315 教室(コンピューター教室) です</p>
+  </section>
+  <section>
+    <h3>指導者メッセージ</h3>
+    <p>
+      プログラミングは楽しい！<br />
+      自分の作ったソフトが誰かの役に立つ喜びを知って欲しい。<br />
```

Comment:
めちゃ細かい上, もともとが間違っていたかもですが, ここでの “欲しい” は形式動詞なので, ひらがなのほうが良いかもです. (>_<;)

```suggestion
      自分の作ったソフトが誰かの役に立つ喜びを知ってほしい。<br />
```